geetools/ee_profiler.py

```diff
@@ -92,12 +92,20 @@ def _to_dict(self, input: str) -> dict:
             else desc,  # Description can have multiple words
         }
 
+        # The default splitting splits the description and only takes
+        # the first word so we need to join the trailing words dropped by the zip.
+        fix_description = False
+        if headers[-1] == "Description":
+            fix_description = True
         # Process each line of data after the header
         for line in lines[1:]:
             # Split the line by spaces, considering multiple spaces as a separator
             # Handle missing values denoted by "-"
             parts = line.split()
             part_result = dict(zip(headers, parts))
+
+            if fix_description:
```

Comment:
the evaluation of the fix_description could actually be done directly in the if statement with the comment associated so it's clear it's only necessary there.